src/drive/ducks/upload/index.js

```diff
@@ -73,8 +71,7 @@ const processNextFile = (
     dispatch({ type: UPLOAD_FILE, file })
     const uploadedFile = await cozy.client.files.create(file, { dirID })
     dispatch({ type: RECEIVE_UPLOAD_SUCCESS, file })
-    // TODO: is the extractFileAttributes call really necessary?
-    fileUploadedCallback(extractFileAttributes(uploadedFile))
+    fileUploadedCallback(uploadedFile)
```

Comment:
Turns out `extractFileAttributes ` is not necessary because the callback actually does it.